docs/changelog/CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 
 ## tip
 
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): change the `query_range` request method from `GET` to `POST`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/6288).
```

Comment:
Does it mean that /query request will remain GET?

Reply:
No, the `/query` request has also been changed to POST. I've updated the changelog to reflect this correction.